bashflux/http.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/mainflux/mainflux"
+)
+
+const (
+	envCertFile = "MF_CERT_FILE"
+	envKeyFile  = "MF_KEY_FILE"
+	envCaFile   = "MF_CA_FILE"
+)
+
+var (
+	httpClient = &http.Client{}
+	serverAddr = "https://0.0.0.0"
```

Comment:
Is this a valid address? Should this address be configurable?

Reply:
It's configurable with --host flag but maybe we can put "localhost" instead.

Reply:
I'd prefer `localhost`.